packages/common/container-definitions/api-report/container-definitions.legacy.alpha.api.md

```diff
@@ -204,17 +204,13 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
     readonly active: boolean;
     readonly clientDetails: IClientDetails;
     readonly hasCheckpointSequenceNumber: boolean;
-    // @deprecated
-    readonly inbound: IDeltaQueue<T>;
```

Comment:
what is the impact on Loop here? I remember there was some usage.

Reply:
They removed that usage already.